File: src/main/java/net/cydhra/nidhogg/exception/UserMigratedException.java
package net.cydhra.nidhogg.exception;

/**
 * Thrown when attempting to log into an Yggdrasil account that is migrated (requires login with e-mail) with the player name.
 */
public class UserMigratedException extends RuntimeException {
    public UserMigratedException(final String message) {
        super(message);
    }
}

File: src/commonMain/kotlin/net/cydhra/nidhogg/exception/UserMigratedException.kt
package net.cydhra.nidhogg.exception

/**
 * Returned by Yggdrasil API server, if the account is migrated, but the provided username was not an email address but
 * the account's username.
 */
class UserMigratedException(message: String) : ForbiddenOperationException(message)